cli/src/command/account.rs

```diff
@@ -283,6 +283,46 @@ pub async fn burn_nft_command(account_handle: &AccountHandle, nft_id: String) ->
 pub async fn balance_command(account_handle: &AccountHandle) -> Result<(), Error> {
     println_log_info!("{:#?}", account_handle.balance().await?);
 
+    let balance = account_handle.balance().await?;
+
+    let current_time = iota_sdk::utils::unix_timestamp_now().as_secs() as u32;
+
+    if balance.potentially_locked_outputs().is_empty() {
+        println_log_info!("no locked outputs");
```

Comment:
This would be an empty [] in the balance overview already, no need to print again?